tests/test_scheduled_app.py

```diff
@@ -16,32 +16,19 @@ def test_set_completed_status():
     event = [
         [
             {
-                "cron_string": "",
-                "environment": "",
-                "app": 0,
-                "app_key": "",
-                "app_connection_id": 0,
-                "app_stream_id": 0,
-                "source_type": "",
-                "company": 0,
-                "provider": "",
+                "app_connection": 0,
+                "app_stream": 0,
                 "schedule": 0,
                 "interval": 0,
-                "schedule_start": "1970-01-01T00:00:00",
-                "schedule_end": "1970-01-01T00:00:00",
+                "schedule_start": 0,
```

Comment:
These are not good starting values. Can we use a real timestamp?

Reply:
the `schedule_start` and `schedule_end` is of type `int` in event, maybe it's be better to use same types when testing as real ones